doc/api/deprecations.md

```diff
@@ -1274,14 +1335,28 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: Runtime
+
+The [`util.log()`][] API has been deprecated. Instead,
```

Comment:
```suggestion
The [`util.log()`][] API has been deprecated because it's an unmaintained
legacy API that was exposed to user land by accident. Instead,
```